@noun-auction/components/AuctionUi/OGNounsBidForm.tsx

```diff
@@ -0,0 +1,88 @@
+import { useAccount, useContractWrite, usePrepareContractWrite, useSigner } from 'wagmi'
+
+import React, { useCallback, useMemo, useState } from 'react'
+import { TypeSafeNounsAuction } from 'validators/auction'
+
+import { NounsBidFormProps, lilNounsAbi, nounsAbi } from '@noun-auction'
+import {
+  LIL_NOUNS_AUCTION_ADDRESS,
+  NOUNS_AUCTION_ADDRESS,
+  createBidAbiFragment,
+} from '@noun-auction/constants/nounish-markets'
+import { PrintError, formatContractError } from '@shared'
+
+import { NounsBidFormComponent } from './NounsBidFormComponent'
+
+export const OGNounsBidForm = ({
+  activeAuction,
+  ...rest
+}: Omit<NounsBidFormProps, 'collectionAddress'> & {
+  activeAuction: TypeSafeNounsAuction
+}) => {
+  const { address } = useAccount()
+  const [bidAmount, setBidAmount] = useState('0')
+
+  let abi =
```

Comment:
This is also being done in SettleAuctionOGNouns. Let's move into a hook or function?

Reply:
+